main/src/ca/uwaterloo/flix/api/Bootstrap.scala

```diff
@@ -605,7 +605,6 @@ class Bootstrap(val projectPath: Path, apiKey: Option[String]) {
     build() flatMap {
       compilationResult =>
         Tester.run(Nil, compilationResult)
-        ().toOk
```

Comment:
This seems ok, but I think we also need to do something in `SimplerRunner` (?)

Reply:
Adding an `exit(1)` in case of a test failure is not a problem.

But:
The `--test` or `--benchmark` arguments should enter one of these branches
https://github.com/flix/flix/blob/master/main/src/ca/uwaterloo/flix/tools/SimpleRunner.scala#L115-L121
but never reach that part because the program enters this branch
https://github.com/flix/flix/blob/master/main/src/ca/uwaterloo/flix/tools/SimpleRunner.scala#L66
and exits afterwards.
So ` java -jar build/libs/flix.jar --test` and ` java -jar build/libs/flix.jar --benchmark` don't work at all now. 

Adding additional checks to
https://github.com/flix/flix/blob/master/main/src/ca/uwaterloo/flix/tools/SimpleRunner.scala#L66
makes them working (again?).

Reply:
See https://github.com/flix/flix/pull/5855/commits/ae31aad5881ca9f8b065aa381c821fb6118ec598